Cannot run the app up. Error: Module not found: Can't resolve '../aws-exports'.

<details>
  <summary>The log</summary>

```
> yarn start
yarn run v1.22.5
$ yarn db:seed:dev
$ ncp ./data/database-seed.json ./data/database.json
$ cross-env NODE_ENV=development concurrently yarn:start:react yarn:start:api
$ yarn tsnode --files backend/app.ts
$ react-scripts -r @cypress/instrument-cra start
$ nyc --silent ts-node -P tsconfig.tsnode.json --files backend/app.ts
[start:api] Error: Cannot find module '../src/aws-exports'
[start:api] Require stack:
[start:api] - D:\Desktop\cypress-realworld-app\backend\helpers.ts
[start:api] - D:\Desktop\cypress-realworld-app\backend\user-routes.ts
[start:api] - D:\Desktop\cypress-realworld-app\backend\app.ts
[start:api]     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:925:15)
[start:api]     at Function.Module._load (node:internal/modules/cjs/loader:769:27)
[start:api]     at Module.require (node:internal/modules/cjs/loader:997:19)
[start:api]     at require (node:internal/modules/cjs/helpers:92:18)
[start:api]     at Object.<anonymous> (D:\Desktop\cypress-realworld-app\backend\helpers.ts:4:61)
[start:api]     at Module._compile (node:internal/modules/cjs/loader:1108:14)
[start:api]     at Module.replacementCompile (D:\Desktop\cypress-realworld-app\node_modules\append-transform\index.js:60:13)
[start:api]     at Module.m._compile (D:\Desktop\cypress-realworld-app\node_modules\ts-node\src\index.ts:1056:23)
[start:api]     at module.exports (D:\Desktop\cypress-realworld-app\node_modules\default-require-extensions\js.js:7:9)
[start:api]     at D:\Desktop\cypress-realworld-app\node_modules\append-transform\index.js:64:4
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
[start:api] yarn run start:api exited with code 1
[start:react] [HPM] Proxy created: [ '/login', '/callback', '/logout', '/checkAuth' ]  ->  http://localhost:3001
[start:react] [HPM] Subscribed to http-proxy events:  [ 'error', 'close' ]
[start:react] i ｢wds｣: Project is running at http://172.29.176.1/
[start:react] i ｢wds｣: webpack output is served from
[start:react] i ｢wds｣: Content not from webpack is served from D:\Desktop\cypress-realworld-app\public
[start:react] i ｢wds｣: 404s will fallback to /
[start:react] Starting the development server...
[start:react]
[start:react] Browserslist: caniuse-lite is outdated. Please run:
[start:react] npx browserslist@latest --update-db
[start:react] Failed to compile.
[start:react]
[start:react] ./src/containers/AppCognito.tsx
[start:react] Module not found: Can't resolve '../aws-exports' in 'D:\Desktop\cypress-realworld-app\src\containers'
```
</details>

--- a/package.json
+++ b/package.json
@@ -150,6 +150,7 @@
     "tsnode": "nyc --silent ts-node -P tsconfig.tsnode.json",
     "tsnode:not-instrumented": "ts-node -P tsconfig.tsnode.json",
     "db:seed": "yarn tsnode scripts/generateSeedData",
+    "predb:seed:dev": "yarn predev:cognito:ci",
     "db:seed:dev": "ncp ./data/database-seed.json ./data/database.json",
     "db:seed:empty": "ncp ./data/empty-seed.json ./data/database.json",
     "postdb:seed": "yarn db:seed:dev",

